website/content/docs/v0.9/Guides/disk-encryption.md

```diff
@@ -0,0 +1,177 @@
+---
+title: "Disk Encryption"
+description: "This page contains guides related kernel level disk encryption"
```

Comment:
Guide on enabling system  disk encryption

?

Reply:
uuh, yeah, I messed up here

Reply:
but at the same time it's not only about enabling the disk encryption, but contains other use cases.